assets/src/edit-story/app/index.js

```diff
@@ -70,16 +70,18 @@ function App({ config }) {
                             <AutoSaveHandler />
                             <TransformProvider>
                               <DropTargetsProvider>
-                                <PrepublishChecklistProvider>
-                                  <GlobalStyle />
-                                  <DevTools />
-                                  <DefaultMoveableGlobalStyle />
-                                  <CropMoveableGlobalStyle />
-                                  <ModalGlobalStyle />
-                                  <CalendarStyle />
-                                  <KeyboardOnlyOutlines />
-                                  <Layout />
-                                </PrepublishChecklistProvider>
+                                <LayoutProvider>
+                                  <PrepublishChecklistProvider>
```

Comment:
This begs the question: why is `PrepublishChecklistProvider` defined up here, and not for example within `Workspace` (`assets/src/edit-story/components/workspace/index.js`)?

Reply:
I'll move it.